database/mysql-driver/dune

```diff
@@ -3,7 +3,6 @@
   (public_name links-mysql)
   (synopsis "MySQL database backend for Links")
   (optional)
-  (flags (:standard -safe-string -dtypes -w Ae-44-45-60 -g -cclib -lunix -thread))
```

Comment:
Does removing these flags mean that they are inherited from the toplevel `dune` file or are the flags just unnecessary in general here?

Reply:
Both. It inherits the flags from root `dune`, the other flags are unnecessary for the package.